### ISSUE TITLE
Errors building the Demo in VSCode

### ISSUE THREAD

**Issue by brianmichalk1** [gephi/gephi-toolkit-demos#13]
I need to display dynamic graphs in my application, and looking at Gephi as a solution.  I cloned the demos, but am getting the following errors:
```
"message": "The project was not built since its build path is incomplete. Cannot find the class file for java.lang.Double. Fix the build path then try building this project",
Type java.lang.Double is indirectly referenced from required .class files but cannot be resolved since the declaring package java.lang exported from module java.base conflicts with a package accessible from module <unnamed>
The type java.lang.Double cannot be resolved. It is indirectly referenced from required .class files
```

**Comment by zhangxiaoxing** [comment]
VSCode is not the most popular Java development tool. Did you followed the [tutorial ](https://code.visualstudio.com/docs/java/java-tutorial) and succeeded in building a few less-demanding  java projects?

**Comment by brianmichalk1** [comment]
Yes, I have been developing java apps in VSCode for about five years now ever since I ditched Eclipse.
Can someone verify that they can compile this maven project?  I suspect that it is a NetBeans dependency, but I don't know much about NetBeans.

**Comment by zhangxiaoxing** [comment]
Earlier gephi depends on outdated netbeans6 framework library, which did cause errors if not set up carefully. I think they updated the GUI part recently, but forgot the toolkit part. I never saw conflict with java.lang.Double before, which must be close to the first line in the code. Is there any classpath-like options you can tinker with?

**Comment by zhangxiaoxing** [comment]
I just built the project freshly cloned from github with jdk11 and NetBeans12 with no problem. I guess it's still a project option thing. JDK11 seems to be required.

**Comment by brianmichalk1** [comment]
It's a NetBeans dependency.
I built the project in Maven, but for integration into my project, I must install the NetBeans plugin, which conflicts with my Extension Pack for Java.  I did get the headless examples to run, but all of my familiar tools are gone, so for me, this is a deal breaker.
Since this is a minimal part of my project, really only useful for evaluating during machine learning, I'll try to get along with the defunct GraphStream libraries.
